release-highlights/release-highlights-2022.15/release-highlights.md

```diff
@@ -0,0 +1,42 @@
+# PatternFly Release Highlights
+## Release 2022.15
+----------------------------------------------------------
+## New features and enhancements
+
+### [File upload - allow customization of progress message](https://www.patternfly.org/v4/components/file-upload---multiple)
+
+![File upload showing error status](./img/file-upload-error.png)
+
+For multiple file uploads, this gives consumers the ability to customize the progress message that displays to provide better error feedback. This is accomplished by adding the capability to add custom helper text below the progress component.
+
+### [Popover help for Tabs](https://www.patternfly.org/v4/components/tabs#help-action)
+
+![tabs with pop-up help](./img/tabs-with-help.png)
+
+Adds the ability to open a help popover from a tab to provide more information about its contents.
+
+### Data sync and data source icons
```

Comment:
FYI - this icon was added as `data-sink` rather than `data-sync` ([issue link](https://github.com/patternfly/patternfly/issues/5226)), should we create a new issue to rename it or change the spelling here?

Reply:
Actually data-sink is correct. Will fix this.